package.json

```diff
@@ -12,12 +12,13 @@
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-icons": "^3.11.0",
     "react-scripts": "^3.4.3",
     "styled-components": "^5.1.1"
   },
   "devDependencies": {
     "@babel/core": "^7.11.1",
+    "@iconify/icons-bx": "^1.0.3",
```

Comment:
so the icon library will be iconify ?

Reply:
Yes, there is an issue resizing the icon with react-icons